# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/plugins/base/nsPluginHost.cpp
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c dom/plugins/base/nsPluginHost.cpp
--- a/dom/plugins/base/nsPluginHost.cpp	Tue Feb 12 17:58:19 2019 +0100
+++ b/dom/plugins/base/nsPluginHost.cpp	Tue Feb 12 16:08:25 2019 +0000
@@ -3160,7 +3160,7 @@
       NS_ENSURE_SUCCESS(rv, rv);
     }
   }
-  rv = channel->AsyncOpen2(listenerPeer);
+  rv = channel->AsyncOpen(listenerPeer);
   if (NS_SUCCEEDED(rv)) listenerPeer->TrackRequest(channel);
   return rv;
 }